### PR TITLE
Update to use render-guides

### PR DESCRIPTION
**Pull request by sarahmccarthy123**
Releases:main

### CODE REVIEW

File: Documentation/FileModule/Index.rst [hunk 1]
```diff
@@ -94,5 +94,6 @@ Next steps
 ==========
 
 The next chapters cover configuration and administration tasks that
-require special access privileges as described in :ref:`privileges`.
+require special access privileges as described in :ref:`Access control in the
```
Comment (froemken) [inline Documentation/FileModule/Index.rst:97]: Should we change that to "Permission"? I remember a patch a week ago, where we started to rename "Access" module to "Permission" for TYPO3 13.

File: Documentation/Localization.ru_RU/FileModule/Index.rst [hunk 2]
```diff
@@ -74,5 +74,6 @@
 Следующие шаги
 ==============
 
-В следующих главах рассматриваются задачи конфигурирования и администрирования, требующие специальных привилегий доступа, описанных в :ref:`privileges`.
+В следующих главах рассматриваются задачи конфигурирования и администрирования, требующие специальных привилегий доступа, описанных в
+:ref:`Access control in the backend (users and groups) <t3coreapi:access>`.
```
Comment (froemken) [inline Documentation/Localization.ru_RU/FileModule/Index.rst:78]: "Permission" module?

File: Documentation/FileModule/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/FileModule/Index.rst:97]: Nope, Access contol and the permissions module are two different things. Therefore the chapter in TYPO3 explained still hat that title.

File: Documentation/Localization.ru_RU/FileModule/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Localization.ru_RU/FileModule/Index.rst:78]: Same as in english

**Comment by linawolf** [comment]
I cannot seem to find the file Documentation/guides.xml did you maybe forget to commit it?

**Comment by linawolf** [comment]
It would also be great if you could activate automatic testing: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Migration/Index.html#migrate-to-testing-workflow

**Comment by linawolf** [comment]
I would suggest a manual backport to 12.4 and 11.5

**Comment by linawolf** [comment]
Thanks a lot for these changes! I formatted the guides.xml and did some small changes.

The differences stem from different handling of official manuals as compared to third party extensions. For a third party extension your solution would have been already quite perfect.

**Review by linawolf — APPROVED** [review]

**Pull request merged**